sphinx_gherkindoc/writer.py

```diff
@@ -315,14 +317,24 @@ def steps(steps: List[behave.model.Step], step_format: str = "{}") -> None:
                 text(step.text)
 
     def examples(
-        example_source: Union[behave.model.Scenario, behave.model.Feature],
+        example_source: Union[
+            behave_models.Scenario,
+            pytest_bdd_models.Scenario,
+            pytest_bdd_models.Feature,
+        ],
         example_source_parent: Optional[behave.model.Feature] = None,
     ) -> None:
         tag_sources = [example_source]
         if example_source_parent:
             tag_sources.append(example_source_parent)
         for example in getattr(example_source, "examples", []):
-            section(3, example)
+            if isinstance(example_source, pytest_bdd_models.Feature):
+                # Set feature examples tables at the same section level as scenarios.
+                section_level = 2
```

Comment:
I wish there were a cleaner way to do this `if` but nothing is coming immediately to mind. RR.

Reply:
Yeah. I though about having constants like `FEATURE_LEVEL = 1` and `EXAMPLES_LEVEL = 2`, but I don't think those are really any more helpful than the plain numbers. Not sure if that's where your thoughts were heading. 

Otherwise, yeah I think `isinstance` is probably the best option. We only change the behavior if the source is of a certain type. We could pass in the level as a param and control it in the logic that calls this function, but it'd be the same `if` check.

Reply:
Having slept on this too, I am surprised I ever signed off on having hard coded constants in the first place.
At least having `section_level = FEATURE_EXAMPLE_LEVEL` and `section_level = SCENARIO_EXAMPLE_LEVEL` would help clarify some, but there are cleanups for another time.

Reply:
@dgou I'm pretty sure all those hard-coded constants were in the original script I wrote back at Rackspace and you _might_ not have been involved in the project yet. Not saying we can't make it better, but to at least let you and Ryan off the hook for now 😁